### ISSUE TITLE
Can turn off but not turn on. 

### ISSUE THREAD

**Issue by nerdtechse** [jghaanstra/com.twinkly#2]
My Twinkly that is the older 56 led version. Will turn of with the app but not turn on. The switch just gets back to off mode after i tried in Homey.  Is there anything i could provide you with to solve this like some log or something?

**Comment by jghaanstra** [comment]
What does the command line log tell you when you run the app (`athom app run`) and try to switch it on.

**Comment by nerdtechse** [comment]
Do i need to set any more for just debug log because all i got in log was 
2018-11-25 22:36:30 [log] [TwinklyApp] Initializing Twinkly App ...
Even if i turned it off..

**Comment by jghaanstra** [comment]
I have created a new branch that has extra logging. Could you run this from the command line, try and switch it on trough Homey and if that does not work switch it with the Twinkly smartphone app and let it run for a minute. Then copy paste the log here.

https://github.com/jghaanstra/com.twinkly/tree/debug

**Comment by nerdtechse** [comment]
2018-11-25 22:50:08 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }
2018-11-25 22:50:12 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }
2018-11-25 22:50:18 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }
2018-11-25 22:50:22 [log] [ManagerDrivers] [twinkly] [0] { code: 1104 }
2018-11-25 22:50:22 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }

**Comment by nerdtechse** [comment]
code 1104 come when i try to turn it on..

**Comment by jghaanstra** [comment]
Please turn it on with the Twinkly app and let the app run on command line so I can see the status when it's on.

**Comment by nerdtechse** [comment]
2018-11-25 22:52:44 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }
2018-11-25 22:52:49 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }
2018-11-25 22:52:54 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }
2018-11-25 22:52:59 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }
2018-11-25 22:53:04 [log] [ManagerDrivers] [twinkly] [0] { mode: 'off', code: 1000 }
2018-11-25 22:53:14 [log] [ManagerDrivers] [twinkly] [0] { mode: 'effect', code: 1000 }
2018-11-25 22:53:19 [log] [ManagerDrivers] [twinkly] [0] { mode: 'effect', code: 1000 }
2018-11-25 22:53:24 [log] [ManagerDrivers] [twinkly] [0] { mode: 'effect', code: 1000 }

**Comment by jghaanstra** [comment]
Interesting, it seems you have a different API. Have you updated your Twinkly to the latest available firmware and could you let me know which version that is.

**Comment by nerdtechse** [comment]
My version is 2.0.23-mqtt. I have some problem before to get twinkly to work with my phones so this could be a beta firmware.

**Comment by jghaanstra** [comment]
I have added more debug logging in the debug branch to retrieve your device info. Could you re-pair the device while running the app from command line and post the output here. If you know how to make http requests with Postman you can also do it with Postman based on this documentation: https://xled.readthedocs.io/en/latest/rest_api.html#device-details

**Comment by jghaanstra** [comment]
> My version is 2.0.23-mqtt. I have some problem before to get twinkly to work with my phones so this could be a beta firmware.

Yes, that version does not seem to be available to me yet. I'm running 2.0.22-mqtt. Perhaps they have changed the API. This shit can happen when using an unofficial API.

**Comment by nerdtechse** [comment]
Are you on Athoms community slack?

**Comment by jghaanstra** [comment]
Yeah, I'm Phuturist there but I'm calling it a night for now.

**Comment by jghaanstra** [comment]
How where you able to get this beta firmware?

**Comment by Emperorneo** [comment]
I'm having the exact same problem but i'm running 2.0.22-mqtt as firmware version. I've got 2 joined 175 led twinkly's. Turning off works immediately.

**Comment by jghaanstra** [comment]
> Edit: I'm still quite new with Homey so I still have to find out how to use the command line to get the logs, but I'm not at home atm.

The link below may help. I really need this info to fix the issue. It looks like different types of strings use different commands. I need to know the differences to be able to fix it.

https://community.athom.com/t/how-to-cli-install-method/198

**Comment by jghaanstra** [comment]
I have submitted a new release (1.0.1) that might fix something. Also, when controling Twinkly with Homey make sure you have the mobile app closed as it might interfere with the token needed to control Twinkly.

**Comment by Emperorneo** [comment]
Strange thing is that we didn't get the update yet, but my on function suddenly started working.
